universal-application-tool-0.0.1/app/views/questiontypes/DropdownQuestionRenderer.java

```diff
@@ -40,6 +40,7 @@ public Tag render(ApplicantQuestionRendererParams params) {
                         toImmutableMap(
                             LocalizedQuestionOption::optionText,
                             option -> String.valueOf(option.id()))));
+    select.setScreenReaderText(question.getQuestionText());
```

Comment:
(question) is setting the screen reader text to the question text for a field the right thing to do?

Reply:
> Looks like name and address question are missing screen reader labels.

Ahh I see. The name and address question fields use labels, so we don't need the screen reader label for them. It would be helpful to comment that in at least the PR description that the multi-field questions have labels attached to the fields, and the single-field questions don't.

Reply:
I _think_ so, yeah.